aem-authoring-toolkit-samples/toolkit-samples-ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/samples/components/content/dungeons-component/dungeons-component.html

```diff
@@ -0,0 +1,15 @@
+<sly data-sly-use.data="com.exadel.aem.toolkit.samples.models.DungeonsComponent"></sly>
+<sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html"></sly>
+<sly data-sly-call="${clientlib.all @ categories='authoring-toolkit.samples.components'}"></sly>
```

Comment:
[Cosmetic] Please make sure HTL options are introduced same way everywhere (and it is better with a whitespace after `@ `)